Makefile.am

```diff
@@ -6694,6 +6698,21 @@ tests += \
 	test-libsystemd-sym \
 	test-libudev-sym
 
+.PHONY: install-tests
+install-tests: $(tests) $(TEST_DATA_FILES)
```

Comment:
Oh, right. I think we *should* install unsafe tests, because they're precisely the kind you want to run in a throw-away build environment, but you are right that this can be a separate PR.